src/main.ts

```diff
@@ -1,8 +1,8 @@
 import * as fs from 'fs'
 import * as core from '@actions/core'
 import * as github from '@actions/github'
-import type {Octokit} from '@octokit/rest'
-import {Webhooks} from '@octokit/webhooks'
+import {GetResponseDataTypeFromEndpointMethod} from '@octokit/types'
```

Comment:
I do not see `@octokit/types` in dependencies, is it working as intended ?

Reply:
It is in dev dependencies and only used for typing

Reply:
surely but in any case we should have it in our package.json ?
By the way, if only used for type -> `import type {GetResponseDataTypeFromEndpointMethod} from '@octokit/types'
`